instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import styled, { css } from "styled-components";
+import {
+  Button,
+  Text,
+  Form,
+  InputField
+} from "../ReusableStyles/ReusableStyles";
+import PropTypes from "prop-types";
+
+const CommentContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 1rem;
+  width: 100%;
+  padding-left: 1rem;
+`;
+
+class CommentSection extends Component {
+  state = {
+    commentsArray: []
+  };
+
+  componentDidMount() {
+    console.log("CDM running");
+    this.setState({
+      commentsArray: this.props.commentsArray
+    });
+  }
+
+  handleChange = event => {
```

Comment:
I would set the localStorage when we're submitting instead of typing, so it doesn't try to set every single time we type for performance benefits